data/tribes/workers/frisians/charcoal_burner/init.lua

```diff
@@ -29,6 +29,10 @@ tribes:new_worker_type {
          "callobject=with_stack",
          "animate=stacking_2 1000",
          "return"
+      },
+      find_pond = {
+         "findobject=attrib:pond_dry radius:8",
+         "return"
```

Comment:
This doesn't need a `return`, the `findobject` alone is enough

Reply:
ok
thanks overlooked that